pilot/pkg/networking/core/v1alpha3/envoyfilter/monitoring.go

```diff
@@ -0,0 +1,57 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package envoyfilter
+
+import (
+	"istio.io/istio/pilot/pkg/features"
+	"istio.io/pkg/monitoring"
+)
+
+var (
+	nodeTag = monitoring.MustCreateLabel("node")
```

Comment:
It is not a good practice to include unbounded labels. I assume the argument would be this is only for debugging and we will only turn it on when needed. Wouldn't it be more convenient to just log these cases?

Reply:
Logging is one option but we want to validate this in a test when a new filter is added in dev/test envs by monitoring these stats.

Reply:
Yeah, we can totally have a metric for this, alert & monitor based on that, but to identify the proxy that does not work with the EnvoyFilter, it would be more reasonable to use log than metric. This way you don't need to have a feature flag to control enablement of this.

Reply:
One of the assumptions I had was we do not generally get in to this issue across all nodes. So the cardinality might not matter. But I can remove the node label.